common/djangoapps/student/management/commands/force_send_notification_digest.py

```diff
@@ -6,28 +6,29 @@
 import pytz
 import logging
 import logging.config
+from django.conf import settings
 
-# This is specifially placed at the top
-# to act as a loggic configuration override for the rest of the
+# This is specifically placed at the top
+# to act as a logic configuration override for the rest of the
 # code
 # Have all logging go to stdout with management commands
 # this must be up at the top otherwise the
 # configuration does not appear to take affect
-logging.config.dictConfig({
-    'version': 1,
-    'handlers': {
-        'console': {
-            'class': 'logging.StreamHandler',
-            'stream': sys.stdout,
+if not getattr(settings, 'TEST_MODE', False):
+    logging.config.dictConfig({
+        'version': 1,
+        'handlers': {
+            'console': {
+                'class': 'logging.StreamHandler',
+                'stream': sys.stdout,
+            }
+        },
+        'root': {
+            'handlers': ['console'],
+            'level': 'INFO'
```

Comment:
(Not super important:)
There has got to be a nicer way to override the logging than doing so at import time. e.g. Subclassing `BaseCommand` and putting a logging override context manager around the `handle` method? This logging override and the `TEST_MODE` fix are both a bit messy.